start-picsure.sh

```diff
@@ -36,6 +36,10 @@ if [ -f $DOCKER_CONFIG_DIR/wildfly/application.truststore ]; then
   export TRUSTSTORE_JAVA_OPTS="-Djavax.net.ssl.trustStore=/opt/jboss/wildfly/standalone/configuration/application.truststore -Djavax.net.ssl.trustStorePassword=password"
 fi
 
+if [ -f $DOCKER_CONFIG_DIR/psama/application.truststore ]; then
+    export PSAMA_TRUSTSTORE_VOLUME="-v $DOCKER_CONFIG_DIR/psama/application.truststore:/usr/local/tomcat/conf/application.truststore"
```

Comment:
is there a scenario where the store doesn't exist where it's still valid for psama to start? Maybe we crash out of the start script